install_deps.sh

```diff
@@ -14,22 +14,22 @@ WGET_FLAGS="--https-only"
 echo ca_certificate=/etc/ssl/certs/ca-certificates.crt > ~/.wgetrc
 
 # Install tools
-apt-get update && apt-get ${APT_GET_FLAGS} install \
+sudo apt-get update && sudo apt-get ${APT_GET_FLAGS} install \
     autoconf \
     automake \
     libtool \
     build-essential \
     ccache \
-    clang-9 \
-    clang-format-10 \
-    clang-tidy-10 \
+    clang \
+    clang-format \
+    clang-tidy \
     curl \
     gdb \
     gdbserver \
     git \
     iptables \
     less \
-    llvm-9 \
+    llvm \
```

Comment:
We are currently supporting llvm-9 so my question is same like it was before for clang